lib/packages/shared-utils/legacy-event-type.ts

```diff
@@ -0,0 +1,56 @@
+import { Action } from "shared-types";
+
+// Resolve the action type based on the GSI1pk
+export const getLegacyEventType = (GSI1pk: string) => {
```

Comment:
```suggestion
export const getLegacyEventType = (gsi1pk: string) => {
```

Variables are lowercase